src/static-layout.js

```diff
@@ -14,13 +14,7 @@
  * limitations under the License.
  */
 
-import {
-  devAssert,
-  devAssertNumber,
-  devAssertString,
-  userAssert,
-} from './core/assert';
-import {transparentPng} from './core/dom/img';
```

Comment:
Nit: We can remove `transparentPng` from `#core/dom/img`, too.

Reply:
Removed, good call.